web/libs/datamanager/src/components/MainView/GridView/GridView.jsx

```diff
@@ -92,14 +134,18 @@ export const GridView = observer(({ data, view, loadMore, fields, onChange, hidd
   const fieldsData = useMemo(() => {
     return prepareColumns(fields, hiddenFields);
   }, [fields, hiddenFields]);
+  const hasImage = fieldsData.some((f) => f.currentType === "Image");
 
-  const rowHeight = fieldsData
-    .filter((f) => f.parent?.alias === "data")
-    .reduce((res, f) => {
-      const height = (DataGroups[f.currentType] ?? DataGroups.TextDataGroup).height;
+  const rowHeight = hasImage
+    ? fieldsData
+        .filter((f) => f.parent?.alias === "data")
+        .reduce((res, f) => {
+          const height = (DataGroups[f.currentType] ?? DataGroups.TextDataGroup).height;
 
-      return res + height;
-    }, 16);
+          return res + height;
+        }, 16)
```

Comment:
What is 16? Why do we add it to all heights?

Reply:
if only i remembered. this was here since forever and i don't remember why it's 16